### DISCUSSION TITLE
Welcome to luci-app-homeproxy Discussions!

### DISCUSSION THREAD

**Discussion by muink** [muink/luci-app-homeproxy#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LoogCP** [comment]
各位大佬，请问怎么做BT和广告的分流
我感觉homeproxy对BT不是很友好，同样是代理所有端口，使用passwallBT速度就很快，但是homeproxy速度不行，甚至有些都不能下载

**Reply by muink** [reply]
用bt为什么要开启所有端口
任何进入sing-box再直连出来的流量都会回落至NAT3
其他peer自然连不进来 只能你连别人
所以别把所有端口开启好吗

**Reply by LoogCP** [reply]
我家的网络本来就是NAT🤣
我以前用passwall都是直接代理所有端口，不知道sing-box有这个差异，
passwall可以直接基于torrent分流，走特定节点，homeproxy不知道行不行，没去研究过
还有就是，不代理所有端口，我可能连tracker都无法访问，这怎么搞

**Reply by muink** [reply]
如果机场线路支持NAT1 并且 hp 开启端点独立映射的话
bt流量走机场是不会NAT降级的
可以在bt client里设定 通过代理服务器连接 tracker

**Reply by muink** [reply]
另外请新开讨论 此线程是公告
